code/API_definitions/qod-api.yaml

```diff
@@ -325,20 +328,54 @@ components:
         * `QOS_S` - Qualifier for the requested QoS profile _S_
         * `QOS_M` - Qualifier for the requested QoS profile _M_
         * `QOS_L` - Qualifier for the requested QoS profile _L_
+    
     Notification:
       type: object
       required:
         - sessionId
-        - event
+        - eventReports
       properties:
         sessionId:
           $ref: "#/components/schemas/SessionId"
-        event:
-          $ref: "#/components/schemas/SessionEvent"
-    SessionEvent:
+        eventReports:
+          type: array
+          items:
+            $ref: "#/components/schemas/EventReport"
+          minItems: 1
+          description: Contains the reported event and applicable information
+    
+    EventReport:
+      type: object
+      required:
+        - eventType
+      properties:
+        eventType:
+          $ref: "#/components/schemas/SessionEventType"
+      discriminator:
+        propertyName: eventType
+        mapping:
+          QOS_STATUS_CHANGED: QosStatusChangedEventReport
```

Comment:
I believe this mapping is incorrect. If this yaml is opened in Swagger, it reports 'QosStatusChangedEventReport' is unused